### ISSUE TITLE
Compute 2D drift/jitter for JWST imaging or spectroscopy

### ISSUE THREAD

**Issue by iancrossfield** [kevin218/Eureka#158]
E.g., adapt the existing Stage 3 code (that computes a 2D drift/jitter for HST data --see calcDrift2D in hst_scan.py) so that we can get these same diagnostics for JWST data as well.  Sooner or later, someone will probably want it!

**Comment by kevin218** [comment]
Will need to add `correct_drift2D` function to the instrument python files (e.g. nirspec.py).

**Comment by taylorbell57** [comment]
I agree that this would definitely be good to have, but as of now none of the simulated data has pointing jitter or drift, so I don't think that this should be a priority for the time being. I am hoping to make a MIRI/LRS simulation with pointing jitter at some point, so that could be a good opportunity to develop and test 2D drift/jitter correction for JWST instruments. That won't be until after the data challenge though

**Comment by kevin218** [comment]
The NIRSpec data have jitter because they are based on actual CV3 observations. My bet is this jitter is why optimal spectral extraction with the median frame isn't working well.

**Comment by taylorbell57** [comment]
Oh interesting, I hadn't appreciated that!
